CustomPlugin/CpController.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using CustomPlugin.Environment;
+using Microsoft.AspNetCore.Mvc;
+
+namespace CustomPlugin
```

Comment:
Should be ServiceLibrary (or MiaServiceLibrary) @davidebianchi what do you think would be best?

Reply:
ServiceLibrary? It is a very common name, but this library could be used everywhere